### PR TITLE
Add code samples for sub-setting methods

### PR DESCRIPTION
**Pull request by irenejoeunpark**
Issue #269 
Added/updated code samples for sub-setting methods

Added ```reset ``` samples and updated  ```update``` samples for the following sub-setting methods.
- [x] Synonyms
- [x] Stop-words
- [x] Searchable attributes
- [x] Displayed attributes
- [x] Attributes for faceting
- [x] Distinct attribute

### CODE REVIEW

**Comment by curquiza** [comment]
Thanks for your PR @irenejoeunpark 
@alallema is on holiday right now and will review your PR once she comes back :)

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -139,28 +135,24 @@ update_ranking_rules_1: |-
...
 get_distinct_attribute_1: |-
   client.index("shoes").getDistinctAttribute();
 update_distinct_attribute_1: |-
-  Settings settings = new Settings();
-  settings.setDistinctAttribute("skuid");
-  client.index("shoes").updateSettings(settings);
+  client.index("shoes").updateDistinctAttributeSettings("skuid");
 reset_distinct_attribute_1: |-
-  //Not yet implemented
+  client.index("movies").resetDistinctAttributeSettings();
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:144]:
```suggestion
  client.index("shoes").resetDistinctAttributeSettings();
```

File: .code-samples.meilisearch.yaml [hunk 2]
```diff
@@ -139,28 +135,24 @@ update_ranking_rules_1: |-
...
+  client.index("movies").updateSearchableAttributesSettings(new String[]
   {
     "title",
     "description",
     "genre"
   });
-  client.index("movies").updateSettings(settings);
 reset_searchable_attributes_1: |-
-  //Not yet implemented
+  clinet.index("movies).resetSearchableAttributesSettings();
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:155]:
```suggestion
  client.index("movies").resetSearchableAttributesSettings();
```

File: .code-samples.meilisearch.yaml [hunk 3]
```diff
@@ -380,9 +368,7 @@ faceted_search_facets_distribution_1: |-
     new SearchRequest("Batman").setFacetsDistribution(new String[] {"genres"});
   client.index("movies").search(searchRequest);
 faceted_search_walkthrough_filterable_attributes_1: |-
-  Settings settings = new Settings();
-  settings.setFilterableAttributes(new String[] {"director", "producer", "genres", "production_companies"});
-  client.index("movies").updateSettings(settings);
+  client.index("movies").updateFilterableAttributesSettings(new String[] {"director", "producer", "genres", "production_companies"});
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:371]:
```suggestion
  client.index("movies").updateFilterableAttributesSettings(new String[]
  {
    "director",
    "producer",
    "genres",
    "production_companies"
  });
```

File: .code-samples.meilisearch.yaml [hunk 4]
```diff
@@ -366,9 +356,7 @@ getting_started_search_md: |-
 
   [About this SDK](https://github.com/meilisearch/meilisearch-java)
 faceted_search_update_settings_1: |-
-  Settings settings = new Settings();
-  settings.setFilterableAttributes(new String[] {"director", "genres"});
-  client.index("movies").updateSettings(settings);
+  client.index("movies").updateFilterableAttributesSettings(new String[] {"director", "genres"});
```
Comment (alallema) [inline .code-samples.meilisearch.yaml:359]:
```suggestion
  client.index("movies").updateFilterableAttributesSettings(new String[]
  {
    "director",
    "genres"
  });
```

**Review by alallema — COMMENTED** [review]
Thanks for this PR @irenejoeunpark!
I propose some fixes and also if you want to update the `update_ranking_rules_1`, here is the code:
```
  client.index("movies").updateRankingRuleSettings(new String[]
  {
    "words",
    "typo",
    "proximity",
    "attribute",
    "sort",
    "exactness",
    "release_date:asc",
    "rank:desc"
  });
```

**Comment by alallema** [comment]
I forgot to say that all the get sub-settings samples should be updated too. They have all been renamed with Settings at the end.
For example:
```
client.index("movies").getRankingRules();
```
became:
```
client.index("movies").getRankingRuleSettings();
```

**Review by alallema — APPROVED** [review]

**Comment by alallema** [comment]
bors merge

**Pull request merged**